RELEASING.md

```diff
@@ -0,0 +1,25 @@
+# Release Guide For Team Members
+
+## Process
+
+1. Merge any changes you want to include into `main`.
+2. Run
+   the [`draft-release`](https://github.com/abcxyz/abc-updater/actions/workflows/draft-release.yml)
+   workflow. Run from the main branch. Choose what sort
+   of [semantic versoning](https://semver.org)
+   increment you want to make (Major, Minor, Patch, Prerelease).
+3. Approve the `PR` created by `draft-release`. Once `PR` is merged
+   the `release`
+   workflow will automatically run.
+4. Optional: Edit description at https://github.com/abcxyz/abc-updater/releases
+
```

Comment:
Maybe add a process for reverting the version on failed releases. If a release fails, the workflow should delete it so the user needs to decrement the version file as you previously did.

Reply:
Couldn't you just re-run the failed release job?

I had a special case where the failed job was invalid so I couldn't re-run it.

Reply:
Wouldnt the version file be incremented so you would skip that version right?

Reply:
Put some language in there

Reply:
> Wouldnt the version file be incremented so you would skip that version right?

The increment happens in the pre-release step, so retrying release shouldn't change anything